gix/src/repository/blame.rs

```diff
@@ -0,0 +1,43 @@
+#[cfg(feature = "blame")]
```

Comment:
Since the whole module is behind a feature toggle, that toggle doesn't need to be used inside of it anymore.